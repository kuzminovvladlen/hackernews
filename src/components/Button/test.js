import React from 'react';
import ReactDOM from 'react-dom';

import renderer from 'react-test-renderer';

import { Button } from './index';


describe('Button', () => {
  it('отрисовывает без ошибки', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Button>Дай мне больше</Button>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  test('есть корректный снимок', () => {
    const component = renderer.create(
      <Button>Дай мне больше</Button>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
