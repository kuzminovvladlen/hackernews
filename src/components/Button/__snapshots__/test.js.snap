// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button есть корректный снимок 1`] = `
<button
  className=""
  type="button"
>
  Дай мне больше
</button>
`;
